aiobiketrax/api.py

```diff
@@ -210,16 +211,30 @@ async def get_trips(
             A list of trips.
         """
         response = await self._get(
-            "trips",
+            "reports/trips",
```

Comment:
this is taken from here: https://app.swaggerhub.com/apis/PowUnity/powunity-biketrax/4.8#/default/get_reports_trips

Reply:
same for the option name below and the date formatting.

Reply:
Awesome that you found the API specification! That's really useful.